frontend/src/scenes/feature-flags/FeatureFlagSettings.tsx

```diff
@@ -39,10 +41,60 @@ export function FeatureFlagSettings({ inModal = false }: FeatureFlagSettingsProp
                     .
                 </p>
             </div>
+            <div className="space-y-2">
+                <FlagDefinitionsAPIKey />
+            </div>
         </div>
     )
 }
 
+export function FlagDefinitionsAPIKey(): JSX.Element {
+    const { currentTeam, isTeamTokenResetAvailable } = useValues(teamLogic)
+    const { resetSecretToken } = useActions(teamLogic)
+
+    const openResetProjectSecretApiKeyDialog = (): void => {
+        LemonDialog.open({
+            title: 'Reset Flag Definitions API key?',
+            description: 'This will invalidate the current Flag Definitions API key and cannot be undone.',
+            primaryButton: {
+                children: 'Reset',
+                type: 'primary',
+                onClick: resetSecretToken,
+            },
+            secondaryButton: {
+                children: 'Cancel',
+                type: 'secondary',
+            },
+        })
+    }
+
+    return (
+        <>
+            <h2 id="secret-api-key" className="min-w-[25rem]">
+                Flag Definitions API key
+            </h2>
+            <p>
+                Use this key to retrieve feature flag definitions for{' '}
+                <Link to="https://posthog.com/docs/feature-flags/local-evaluation">local evaluation</Link>.
+            </p>
+            <CodeSnippet
+                actions={
+                    isTeamTokenResetAvailable ? (
+                        <LemonButton icon={<IconRefresh />} noPadding onClick={openResetProjectSecretApiKeyDialog} />
+                    ) : undefined
+                }
+                thing="Flag Definitions API key"
+            >
+                {currentTeam?.secret_api_token || ''}
```

Comment:
can you add a default value for this to `api.mock.ts`, similar to this one? https://github.com/PostHog/posthog/blob/master/frontend/src/lib/api.mock.ts#L48. 

It'll help the stories render better for the snapshot tests.